### DISCUSSION TITLE
Upgrade the source to HTTPS, but the native binding is still HTTP, resulting in a Publish Fail

### DISCUSSION THREAD

**Discussion by 1414044032** [verdaccio/verdaccio#2312] [Q&A]
1: server is update use “https”：`https://10.12.12.12` 。
2: local npm is use http: `http://10.12.12.12` npm config set registry http://10.12.12.12。
3: `install` is success 。 but when i `publish` 。verdaccio get a http status `301` It doesn't redirect itself。
4: The NPM client reported no error, but the package could not be uploaded。
5: Why is that, why doesn't he redirect himself
System:
    OS: Linux 3.10 Debian GNU/Linux 9 (stretch) 9 (stretch)
    CPU: (32) x64 Intel(R) Xeon(R) CPU E5-2630 v3 @ 2.40GHz
  Binaries:
    Node: 10.15.0 - /usr/local/bin/node
    Yarn: 1.12.3 - /usr/local/bin/yarn
    npm: 6.4.1 - /usr/local/bin/npm
  npmGlobalPackages:
    verdaccio: 4.11.3

**Comment by juanpicado** [comment]
hi @1414044032 please add more info, be more detailed.
